[DOCS-3138] AWS Troubleshooting (#13089)
DataDog/documentation: content/en/integrations/guide/aws-integration-troubleshooting.md

## Tags

### Hosts still have AWS tags after removing the AWS EC2 integration

You can use the AWS integration to collect data from CloudWatch or install a Datadog Agent directly on each EC2 instance to get data and tags. For several reasons, you may have opted to use both of these methods to collect data. In this case, Datadog's backend merges the data from both from the integration and the Datadog Agent to a single host object.

If you removed the AWS integration, but continue to run a Datadog Agent on your EC2 instances, the hosts in your Datadog account continues to have the old host-tags associated with it that were collected from AWS. This is intended behavior, and it does not indicate that the AWS integration or AWS EC2 integration is still enabled.

You can verify by checking the “Apps Running” for that host via the infrastructure list or checking the metrics summary and creating a notebook scoped to that host.

> If you prefer to have these AWS host tags permanently removed from a host, you can do this by using the [Remove host tags API endpoint][7].

Reviewer: ```suggestion
If you want to permanently remove AWS host tags from a host, you can do this by using the [Remove host tags API endpoint][7].
```
